airbyte-integrations/connectors/source-salesforce/source_salesforce/manifest.yaml

```diff
@@ -0,0 +1,112 @@
+version: 5.17.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - CalendarView
```

Comment:
Outside of the scope of this PR: Specific to Salesforce: do we guarantee that this entity is available to all our users?

Reply:
I’m not sure. This is just a random stream available in our test account. I think we should have a static stream to check the connection or some default entities that are available for all users.